types/common_label.go

```diff
@@ -16,6 +16,24 @@ const (
 	LabelFrontendEntryPoints                     = LabelPrefix + "frontend.entryPoints"
 	LabelFrontendRequestHeader                   = LabelPrefix + "frontend.headers.customrequestheaders"
 	LabelFrontendResponseHeader                  = LabelPrefix + "frontend.headers.customresoponseheaders"
```

Comment:
I just noticed this typo: `customresoponseheaders`

Reply:
It's already fixed in `master`